src/main/java/com/spotify/autoscaler/Autoscaler.java

```diff
@@ -82,13 +74,13 @@ public void run() {
     try {
       runUnsafe();
     } catch (final Exception t) {
-      logger.error("Unexpected Exception!", t);
+      LOGGER.error("Unexpected Exception!", t);
     }
   }
 
   private void runUnsafe() {
     autoscalerMetrics.markHeartBeat();
-
+    ConcurrentHashMap<String, Boolean> hasRun = new ConcurrentHashMap<>();
```

Comment:
do we really need this? I don't see how it can run for the same cluster twice, and even if it does, it's really not that big of a problem (we do have multiple instances of the service anyways)
the `hasRun` in `AutoscaleJob` is a leftover code

Reply:
Yeah, there should be a lock at the database level, right? I guess the `hasRun` was left there as a double check, but it's really ugly. 

We could test somehow that if we call runUnsafe() twice, we autoscale only once? Or something like that? Maybe in another PR. In this one I would just remove the check.

Reply:
ah @gizem969 got it. I was unsure why such a check was there in the first place. I'd be glad to remove it.